<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>Plots - dropSeqPipe</title>
  <link href='https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700' rel='stylesheet' type='text/css'>

  <link rel="stylesheet" href="../css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../css/theme_extra.css" type="text/css" />
  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
  
  <script>
    // Current page data
    var mkdocs_page_name = "Plots";
    var mkdocs_page_input_path = "Plots.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
  
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> dropSeqPipe</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
	<ul class="current">
	  
          
            <li class="toctree-l1">
		
    <a class="" href="..">Home</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../Installation/">Installation</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../Reference-Files/">Reference Files</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../Create-config-files/">Config file and sample file</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../Running-dropSeqPipe/">Running dropSeqPipe</a>
	    </li>
          
            <li class="toctree-l1 current">
		
    <a class="current" href="./">Plots</a>
    <ul class="subnav">
            
    <li class="toctree-l2"><a href="#1-knee-plot-per-sample">1. Knee plot (per sample)</a></li>
    

    <li class="toctree-l2"><a href="#2-cell-barcode-quality-trim-per-sample">2. Cell barcode Quality trim (per sample)</a></li>
    

    <li class="toctree-l2"><a href="#3-umi-quality-trim-per-sample">3. UMI Quality trim (per sample)</a></li>
    

    <li class="toctree-l2"><a href="#4-polya-trimming-of-reads-per-sample">4. PolyA trimming of reads (per sample)</a></li>
    

    <li class="toctree-l2"><a href="#5-smart-adapter-trimming-of-reads-per-sample">5. SMART adapter trimming of reads (per sample)</a></li>
    

    <li class="toctree-l2"><a href="#6-rna-metrics-per-sample">6. RNA metrics (per sample)</a></li>
    

    <li class="toctree-l2"><a href="#7-cell-and-umi-dropped-barcodes-across-samples">7. Cell and UMI dropped Barcodes (across samples)</a></li>
    

    <li class="toctree-l2"><a href="#8-yield-across-samples">8. Yield (across samples)</a></li>
    

    <li class="toctree-l2"><a href="#mixed-experiment">Mixed experiment</a></li>
    
        <ul>
        
            <li><a class="toctree-l3" href="#8-barnyard-plot-per-sample">8. Barnyard plot (per sample)</a></li>
        
        </ul>
    

    </ul>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../CHANGELOG/">Change Log</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../FAQ/">FAQ</a>
	    </li>
          
        </ul>
      </div>
      &nbsp;
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">dropSeqPipe</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>Plots</li>
    <li class="wy-breadcrumbs-aside">
      
        <a href="https://github.com/Hoohm/dropSeqPipe/edit/master/docs/Plots.md"
          class="icon icon-github"> Edit on GitHub</a>
      
    </li>
  </ul>
  <hr/>
</div>
          <div role="main">
            <div class="section">
              
                <p>On of the main purpose of this package is getting information about your data to improve your protocol and filter your data for further downstream analysis.</p>
<p>Here is a list of plots and reports that you will get from the pipeline.</p>
<p>Fastqc, STAR and trimmomatic reports are now generated as <a href="http://multiqc.info/docs/#using-multiqc-reports">multiqc reports</a> in the reports folder.</p>
<h2 id="1-knee-plot-per-sample">1. Knee plot (per sample)</h2>
<p><img alt="Knee plot" src="../images/sample1_knee_plot.png" />
On the x axis is the cumulative fraction of reads per STAMPS (captured cell).<br />
On the y axis is the ordered STAMPS (based on total reads).
This allows you to determine how much of the reads you actually captured with the number of cells you expected.
The cutting is based on the <code>expected_cells</code> parameter in the <code>samples.csv</code> file.
If you see a clear bend on the plot that is higher in the number of cells than what you expected, you should increase the <code>expected_cells</code> value and rerun the <code>extract</code> step. If it is under, I would advise to filter out your data with a downstream analysis tool such as Seurat.
<em>Note: I advise not to try to discover "real" cells/STAMPS at this stage. I suggest to extract the expected number of cells and filter out later in post-processing with other kind of meta data.</em> </p>
<h2 id="2-cell-barcode-quality-trim-per-sample">2. Cell barcode Quality trim (per sample)</h2>
<p><img alt="Cell barcode Quality trim" src="../images/sample1_CELL_dropped.png" />
On the x axis are the number of failed bases in the cell barcodes.
On the y axis are the number of cell barcodes.
This plot allows you to adjust your filter of minimum quality and number of allowed minimum quality for the filtering process of the cell barcodes.</p>
<h2 id="3-umi-quality-trim-per-sample">3. UMI Quality trim (per sample)</h2>
<p><img alt="UMI barcode Quality trim" src="../images/sample1_UMI_dropped.png" />
On the x axis are the number of failed bases in the UMI barcode.
On the y axis are the number of UMI barcodes.
This plot allows you to adjust your filter of minimum quality and number of allowed minimum quality for the filtering process of the UMIs.</p>
<h2 id="4-polya-trimming-of-reads-per-sample">4. PolyA trimming of reads (per sample)</h2>
<p><img alt="PolyA trim" src="../images/sample1_polya_trimmed.png" />
On the x axis are the length of the trimmed polyA<br />
On the y axis are the number of trimmed polyA.<br />
This plot shows you the distribution of the polyA trimming step.</p>
<h2 id="5-smart-adapter-trimming-of-reads-per-sample">5. SMART adapter trimming of reads (per sample)</h2>
<p><img alt="SMART adapter trimming" src="../images/sample1_start_trim.png" />
On the x axis are the length of the trimmed SMART adapter<br />
On the y axis are the number of trimmed SMART adapter.<br />
This plot shows you the distribution of the SMART adapter trimming step.</p>
<h2 id="6-rna-metrics-per-sample">6. RNA metrics (per sample)</h2>
<p><img alt="RNA metrics" src="../images/sample1_rna_metrics.png" />
On the x axis are top barcodes based on your <code>expected_cells</code> values or the <code>barcodes.csv</code> file.<br />
Top plot: On the y axis are the number of bases classified by region of mapping.<br />
Bottom plot: On the y axis are the percentage of bases classified by region of mapping.
This plot gives a lot of different informations. The top plot allows you to quickly compare cells between them in terms of how much has been mapped. This can sometimes help identify outliers or bad runs.
The bottom plot allows you to find cells that have an "abnormal" mapped base distribution compared to other cells.</p>
<h2 id="7-cell-and-umi-dropped-barcodes-across-samples">7. Cell and UMI dropped Barcodes (across samples)</h2>
<p><img alt="BC quality dropped" src="../images/BC_drop.png" />
On the x axis are the samples.<br />
TOP: On the y axis are the number of reads discarded.<br />
BOTTOM: On the y axis are the percentage of reads discarded.<br />
This plot allows you to adjust your filter of minimum quality and number of allowed minimum quality for the filtering process of the UMIs. The <code>Not_dropped</code> category is going to be mapped in STAR.</p>
<h2 id="8-yield-across-samples">8. Yield (across samples)</h2>
<p><img alt="BC quality dropped" src="../images/yield.png" />
On the x axis are the samples.<br />
TOP: On the y axis are the number of reads attributed to each category.
BOTTOM: On the y axis are the percentage of attributed to each category.
This plot gives you an overview of all the reads from your samples and how they are distributed in all the possible categories. The reads that are uniquely mapped ar the ones you will keep at the end for the UMI count matrix.</p>
<h1 id="mixed-experiment">Mixed experiment</h1>
<h2 id="8-barnyard-plot-per-sample">8. Barnyard plot (per sample)</h2>
<p><img alt="Barnyard plot" src="../images/hum_mus_species_plot_transcripts.png" />
This plot shows you species purity for each STAMPS. Mixed and No call STAMPS are dropped and only single species are kept for extraction.
You can change the minimum ratio of transcripts to define a STAMP as mixed or not in the configfile with: <code>species_ratio</code>
You get one plot for genes and one plot for transcripts. The selection is done on the transcript level.</p>
              
            </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="../CHANGELOG/" class="btn btn-neutral float-right" title="Change Log">Next <span class="icon icon-circle-arrow-right"></span></a>
      
      
        <a href="../Running-dropSeqPipe/" class="btn btn-neutral" title="Running dropSeqPipe"><span class="icon icon-circle-arrow-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="http://www.mkdocs.org">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" style="cursor: pointer">
    <span class="rst-current-version" data-toggle="rst-current-version">
      
          <a href="https://github.com/Hoohm/dropSeqPipe/" class="fa fa-github" style="float: left; color: #fcfcfc"> GitHub</a>
      
      
        <span><a href="../Running-dropSeqPipe/" style="color: #fcfcfc;">&laquo; Previous</a></span>
      
      
        <span style="margin-left: 15px"><a href="../CHANGELOG/" style="color: #fcfcfc">Next &raquo;</a></span>
      
    </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>

</body>
</html>
